experimental/dds/tree/package.json

```diff
@@ -41,7 +31,6 @@
 		"build:compile": "fluid-build . --task compile",
 		"build:docs": "api-extractor run --local",
 		"build:esnext": "tsc --project ./tsconfig.esnext.json",
-		"check:are-the-types-wrong": "attw --pack . --exclude-entrypoints ./test/EditLog",
```

Comment:
before your change, I think this is validating all entry points except the one you just removed, and now you have removed the validation. That doesn't seem like the right change.

Reply:
Updated to include validation check again